deploifai/project/info.py

```diff
@@ -0,0 +1,70 @@
+import click
+
+from deploifai.api import DeploifaiAPIError
+from deploifai.context import (
+    pass_deploifai_context_obj,
+    DeploifaiContextObj,
+    is_authenticated, project_found,
+)
+
+
+@click.command()
+@pass_deploifai_context_obj
+@is_authenticated
+@project_found
+def info(context: DeploifaiContextObj):
+    """
+    Command to get the information of a project
+    """
+
+    current_workspace = context.global_config["WORKSPACE"]["username"]
+
+    click.secho("Workspace Name: {}".format(current_workspace), fg="blue")
+
+    project_id = context.local_config["PROJECT"]["id"]
+
+    where_project = {"id": {"equals": project_id}}
+
+    fragment = """
+                fragment project on Project {
+                    name status
+                    cloudProfile{
+                        provider
+                    }
+                    experiments{
+                        name status environment
+                    }
+                    dataStorages{
+                        name status
+                    }
+                    trainings{
+                        name status state
+                    }
+                    applications{
+                        applicationName
+                    }
+                }
+                """
+    projects_data = context.api.get_projects(workspace=current_workspace, where_project=where_project, fragment=fragment)
```

Comment:
Isn't there a `get_project` api call that just gets a `project` in the graphql api?

Reply:
yes will use that instead